Python/optimizer.c

```diff
@@ -372,6 +372,32 @@ static PyTypeObject UOpExecutor_Type = {
     .tp_as_sequence = &uop_as_sequence,
 };
 
+static int
+squeeze_stubs(
```

Comment:
```suggestion
shift_stubs(
```

(the name implied to me that something happens to the size of the stubs).

Reply:
How about `move_stubs()` (echoing that it uses `memmove()`).